dev/benchmarks/complex_layout/lib/main.dart

```diff
@@ -412,7 +412,9 @@ class ItemImageBox extends StatelessWidget {
                   child: new Container(
                     decoration: new BoxDecoration(
                       backgroundColor: Colors.black54,
-                      borderRadius: 2.0
+                      borderRadius: const BorderRadius.all(
```

Comment:
can we have a default constructor on BorderRadius that just takes a double and sets it to all 8 radii? Going from three characters to three lines is sad making. :-(